ps_googleanalytics.php

```diff
@@ -139,11 +139,15 @@ public function hookDisplayAdminOrder()
     {
         $gaTagHandler = new PrestaShop\Module\Ps_Googleanalytics\Handler\GanalyticsJsHandler($this, $this->context);
 
-        echo $gaTagHandler->generate(
+        $output = '';
```

Comment:
@Hlavtox do we really need this empty var here?

Reply:
@kpodemski Maybe not?

If the function somehow fails, is it ok if we return false or whatever?

Reply:
@Hlavtox yes, unless there's a check `if (Hook::exec)` but that's not the case in PrestaShop from what I know :)

Reply:
@Hlavtox you can check this by having `return false;` here and creating other module which is also hooked into `displayAdminOrder`, just to make sure ;)

Reply:
Removed ;-)